src/SpotifyApi.NetCore/IFollowApi.cs

```diff
@@ -93,5 +93,119 @@ public interface IFollowApi
+        /// <summary>
+        /// Follow a Playlist
+        /// </summary>
+        /// <param name="playlistId">Required. The Spotify ID of the playlist. Any playlist can be followed, regardless of its public/private status, as long as you know its playlist ID.</param>
+        /// <param name="isPublic">Optional. Defaults to true. If true the playlist will be included in user’s public playlists, if false it will remain private.</param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/follow-playlist/
+        /// </remarks>
+        Task FollowPlaylist(
+            string playlistId,
+            bool isPublic = true,
+            string accessToken = null
+            );
+        #endregion
+
+        #region GetUsersFollowedArtists
+        /// <summary>
+        /// Get User's Followed Artists
+        /// </summary>
+        /// <param name="limit">Optional. The maximum number of items to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="after">Optional. The last artist ID retrieved from the previous request.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedArtists"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/get-followed/
+        /// </remarks>
+        Task<PagedArtists> GetUsersFollowedArtists(
+            int limit = 20,
+            string after = null,
+            string accessToken = null
+            );
+
+        /// <summary>
+        /// Get User's Followed Artists
+        /// </summary>
+        /// <param name="limit">Optional. The maximum number of items to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="after">Optional. The last artist ID retrieved from the previous request.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedArtists"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/get-followed/
+        /// </remarks>
+        Task<T> GetUsersFollowedArtists<T>(
+            int limit = 20,
+            string after = null,
+            string accessToken = null
+            );
+        #endregion
+
+        #region UnfollowArtistsOrUsers
+        /// <summary>
+        /// Unfollow Artists or Users
+        /// </summary>
+        /// <param name="artistIds">A comma-separated list of the artist Spotify IDs. A maximum of 50 artist IDs can be sent in one request. A minimum of 1 artist id is required.</param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/unfollow-artists-users/
+        /// </remarks>
+        Task UnfollowArtists(
+            string[] artistIds,
+            string accessToken = null
+            );
+
+        /// <summary>
+        /// Unfollow Artists or Users
```

Comment:
This should be "Remove the current user as a follower of one or more other Spotify users." as per docs